services/app/src/components/views/Search.test.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
-import {LanguageType} from 'shared/schema/Constants';
-import {Quest} from 'shared/schema/Quests';
+import {LanguageType} from '../../../../../shared/schema/Constants';
```

Comment:
what's going on here with all of the `../`'s? We should support the `shared/` alias shortcut